### PR TITLE
Added arg 'use-default-git' to select default git account and avoid prompting user

### PR DESCRIPTION
**Pull request by ssathi**
Description

When importing new git projects, jx import command asks to confirm git username. Here I have added a new argument 'use-default-git' to give consent that use knows his/her default account and use. The account is taking from the current pipeline. 

Trying to resolve this issue:
https://github.com/jenkins-x/jx/issues/334

### CODE REVIEW

**Comment by rawlingsj** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/jenkins-x/jx/pull/3206#issuecomment-467301644" title="Approved">jstrachan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=jenkins-x%2Fjx).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/jenkins-x/jx/blob/master/OWNERS)~~ [jstrachan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ssathi** [comment]
@rawlingsj 

Could you please send me the detail error log. Seems like this is failing for others as well.

**Comment by ssathi** [comment]
@jstrachan 
Could you please post me the error logs for this failure?

File: pkg/jx/cmd/import.go [hunk 1]
```diff
@@ -258,12 +260,17 @@ func (options *ImportOptions) Run() error {
 				return err
 			}
 		}
+
+		if options.UseDefaultGit {
+		    userAuth= config.CurrentUser(server, true)
```
Comment (ccojocar) [inline pkg/jx/cmd/import.go]: You should use the `options.InCluster` instead of `true` .

Comment (ssathi) [reply pkg/jx/cmd/import.go]: @ccojocar 

You mean options.CommonOptions.InCluster()?

**Comment by ssathi** [comment]
@ccojocar

Could you please let me what "build step "build-step-jenkins" exited with code 2" means? Could you also post error details please?

**Comment by ccojocar** [comment]
@ssathi There are some formatting issues. Please could you run go fmt over the changes?

**Comment by ssathi** [comment]
@ccojocar
Could you please merge this PR?

**Pull request merged**